src/javaRosa.js

```diff
@@ -1980,6 +1973,24 @@ define([
                 widget: iTextIDWidget,
                 widgetValuePath: "constraintMsgItext"
             };
+            databind.constraintMediaIText = function (mugOptions) {
+                return mugOptions.isSpecialGroup ? undefined : {
```

Comment:
heh, who wrote that code? blame says mwhite, but I didn't dig deeper to see if he copied it from somewhere. It's probably just my personal sense of style, but I don't like to explicitly return `undefined` from a function that is intended to return a value (that's what `null` is for). I can't really think of a valid reason other than something like:
```js
function getValue(arg) {
    return arg < 0 ? undefined :  arg + 1;
}

var props = {value: getValue(-1)}

// is this confusing?
props.value === undefined       // true
props.val === undefined         // true
props.hasOwnProperty("value")   // true
props.hasOwnProperty("val")     // false
```
You can ignore this comment too. It's just me ranting about silly differences between `null` and `undefined`.